### ISSUE TITLE
Make custom.css easier to preserve

### ISSUE THREAD

**Issue by clinkingdog** [Softwire/Workflowy2Web#38]
Although it's nice for the zip to provide an empty custom.css file the first time the prototype is built, it's actually unhelpful afterwards because if it's unzipped over an existing version with stuff added to custom.css, it overwrites that stuff with an empty file - potentially losing a lot of valuable work.

Instead the zip should provide an empty custom.template.css, with text content advising the user to rename it to custom.css in order to use it.

The `<link>` in the HTML `<head>` should then continue to point to custom.css, rather than the template one.

**Comment by usmanatron** [comment]
What about a half-way house where you include a custom.template.css file and instructions to rename it the first time?
That way you get the initial niceness without the damage you point out?
(Happy to have a go at doing this :-) )

**Comment by clinkingdog** [comment]
Yep, that's a nice one - description updated. Another alternative that occurred to me was to change the `<link>` to refer to something outside the zip altogether - but I think your solution is nicer.
